spec/models/accounting/interest_calculator_spec.rb

```diff
@@ -75,43 +75,43 @@
         expect(t3.line_items.size).to eq(2)
 
         # account details
-        expect(t3.line_item_for(int_rcv_acct).amount).to equal_money(2.24)
+        expect(t3.line_item_for(int_rcv_acct).amount).to equal_money (0.08/365 * 27 * 117.50).round(2)
```

Comment:
I'm generally suspicious of using too many "magic numbers"  by themselves -- would there be a way to define these as constants so that it's easier for somebody coming into this later to see what the components going into this calculation are and why they are being done this way?

Reply:
left you a note on slack